migrations/updateUserRolesDocuments/updateUserRolesCodes.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
@Prajwal17Tunerlabs Please include a readme to explain how we can run the script